docs/jax-101/index.rst

```diff
@@ -1,3 +1,5 @@
+:orphan:
```

Comment:
Still needed?

Reply:
Yes because its not in a toctree if I dont add it well get a warning